frontend/src/Utils/plugins.ts

```diff
@@ -0,0 +1,16 @@
+export type PluginType = {
+  name: string;
+  pathPrefix?: string;
+};
+
+type GetActivePlugins = (isBeta: boolean, appName: string) => Promise<PluginType[]>;
+
+export const getActivePlugins: GetActivePlugins = async (isBeta, appName) => {
+  try {
+    return (await fetch(`${isBeta ? '/beta' : ''}/apps/${appName}/plugins.json`)).json();
+  } catch (e) {
+    // eslint-disable-next-line no-console
+    console.error(e);
```

Comment:
🤔 
`axios` has been a topic of conversation at several places I have been... but I am unsure how much it brings to the table over `fetch` with proper promise understanding. I'm not against this `try/catch` solution, so we can go with this change.

However... please do this one change:
```suggestion
    console.error('Failed to fetch plugin data', e);
```
I think all errors should come with a context. An error can often be silly and unhelpful... a prefix string helps locate where the issue came from.

Reply:
The reason why we prefer `axios` over `fetch` in console.dot is that we added some `interceptors` to properly shape the data, dispatch errors if something happend (for instance server down etc.)